frontend/src/utils/helpers.js

```diff
@@ -91,3 +91,17 @@ export const normalizePhone = (value, previousValue) => {
 };
 
 export const upperCase = (value) => value && value.toUpperCase();
+
+export const getFiscalYear = () => {
+  const today = new Date();
+  const currentYear = today.getFullYear();
+  // new Date is index 0, Jan = 0
+  const fiscalYear = new Date(currentYear, 3, 1); // April 1, currentYear
+  if (today > fiscalYear) {
+    return currentYear;
+  }
+  // return the previous year if between the months of Jan-March
+  return currentYear - 1;
+};
+
+export const returnBoolean = (param) => param && param === "true";
```

Comment:
nooooo, nothing to do with our API. 

I had this in mineDashboard as a class method, but decided to move it out.
When filtering the open_orders, one drop down option has true/false values but they were returned as strings after being pushed to the URL, so I needed to transform them back into booleans. 

Have any alternate suggestions?

Reply:
It should be functionally equivalent to `someVariableFromTheUrl === "true"`, right?

The case when that wouldn't be true is if `param` (in this case) is a falsey value other than `false`, which would cause it to short-circuit and return. Such as if `param = ""`, then `returnBoolean` would return `""`.

Hopefully you're not relying on that behaviour, given the name of the function? So, I'd say we can probably just scrap this function and use `===` directly. Thoughts?

Reply:
yep, happy to do that!

Reply:
removed!